index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <!-- Poppins font form Google APIs cdn -->
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <header id="header">
+        <a href="#" id="my_logo">My Logo</a> 
+        <nav>
+            <a href="#" id="img_link">
+                <img src="assets/images/menu.png" id="menu_img" alt="menu">
+            </a>
+        </nav>                
+    </header>
+    <main>              
+        <section id="banner_area">
+            <div id="text-area">
+                <h2 id="name_title">I’m Chima Glad to see you!</h2>
+                <p id="subtitle">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p id="mini_title">LET’S CONNECT</p>
+                <nav id="social_links">
+                    <ul id="social_icons">
+                        <li><img src="assets/images/twitter.svg" alt="Twitter" class="icon_image"></li>
+                        <li><img src="assets/images/linkedin.svg" alt="Linkedin" class="icon_image"></li>
+                        <li><img src="assets/images/medium.svg" alt="Medium" class="icon_image"></li>
+                        <li><img src="assets/images/git.svg" alt="Git" class="icon_image"></li>
+                        <li><img src="assets/images/angelist.svg" alt="Angelist" class="icon_image"></li>
```

Comment:
- [OPTIONAL] KIndly make sure every link is wrapped into the Anchor tag `a` because apart from being semantically correct, the `a ` tag will let you connect your social networks with the icons, using `href` attribute.